src/Model/Behavior/SequenceBehavior.php

```diff
@@ -274,7 +274,7 @@ public function setOrder(array $records)
         $config = $this->config();
         $table = $this->_table;
 
-        $table->removeBehavior('Sequence');
+        $table->removeBehavior('ADmad/Sequence.Sequence');
```

Comment:
I don't think plugin prefix is needed when removing behavior. Probably this might be the cause of failing test.